_maps/RandomRuins/SpaceRuins/crashedship.dmm

```diff
@@ -383,6 +383,7 @@
 /obj/machinery/porta_turret{
 	dir = 8;
 	installation = /obj/item/gun/energy/lasercannon;
+	
```

Comment:
```suggestion
```

introducing some wack ass newline in the middle of the props section of a map key is a good way to instantly die i think